### PR TITLE
[tests-only][full-ci] bump ocis commit id - master

### PR DESCRIPTION
**Pull request by anon-pradip**
Bump ocis commit id to the latest
Part of https://github.com/owncloud/QA/issues/858

Fixes https://github.com/owncloud/web/issues/11373

### CODE REVIEW

**Comment by anon-pradip** [comment]
image
When running the test for file: `tests/e2e/cucumber/features/app-provider/urlJourneys.feature:10`
 for the step:
```feature
Then "Alice" should see the content "Microsoft Word Content" in editor "OnlyOffice"
```
,locally, I get the above popup modal and error : 
```
locator.click: Timeout 30000ms exceeded.
       Call log:
         - waiting for frameLocator('[name="app-iframe"]').frameLocator('[name="frameEditor"]').locator('#id_viewer_overlay')
         -   locator resolved to <canvas width="931" height="486" class="block_elem" id="id_viewer_overlay"></canvas>
         - attempting click action
         -   waiting for element to be visible, enabled and stable
         -   element is visible, enabled and stable
         -   scrolling into view if needed
         -   done scrolling
         -   <div class="footer center">…</div> from <div id="window-view699" class="asc-window modal alert">…</div> subtree intercepts pointer events
         - retrying click action, attempt #1
         -   waiting for element to be visible, enabled and stable
         -   element is visible, enabled and stable
         -   scrolling into view if needed
         -   done scrolling
         -   <div class="footer center">…</div> from <div id="window-view699" class="asc-window modal alert">…</div> subtree intercepts pointer events
         - retrying click action, attempt #2 ........
```
and same error in CI too: https://drone.owncloud.com/owncloud/web/46584/15/14


Hitting that "Ok" button also does not reload the page.

File: tests/e2e/cucumber/steps/ui/resources.ts [hunk 1]
```diff
@@ -926,10 +927,11 @@ Then(
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const resourceObject = new objects.applicationFiles.Resource({ page })
     for (const info of stepTable.hashes()) {
+      const actions = await resourceObject.getAllAvailableActions({ resource })
       if (actionType === 'should') {
-        expect(await resourceObject.getAllAvailableActions({ resource })).toContain(info.action)
+        expect(actions.some((action) => action.includes(info.action))).toBe(true)
```
Comment (saw-jan) [inline tests/e2e/cucumber/steps/ui/resources.ts:932]:
```suggestion
        expect(actions.some((action) => action.startsWith(info.action))).toBe(true)
```

File: tests/e2e/cucumber/steps/ui/resources.ts [hunk 2]
```diff
@@ -926,10 +927,11 @@ Then(
...
       } else {
-        expect(await resourceObject.getAllAvailableActions({ resource })).not.toContain(info.action)
+        expect(actions.some((action) => action.includes(info.action))).toBe(false)
```
Comment (saw-jan) [inline tests/e2e/cucumber/steps/ui/resources.ts:934]:
```suggestion
        expect(actions.some((action) => action.startsWith(info.action))).toBe(false)
```

**Review by saw-jan — APPROVED** [review]

**Review by ScharfViktor — APPROVED** [review]
LGTM

**Pull request merged**